src/main/java/cubicchunks/server/chunkio/IONbtWriter.java

```diff
@@ -54,7 +57,9 @@
 
 @MethodsReturnNonnullByDefault
 @ParametersAreNonnullByDefault
-class IONbtWriter {
+public class IONbtWriter {
+    
+    public static final int MINECRAFT_CHUNK_DATA_VERSION = 819;
```

Comment:
Actually, no test may be needed at all. Why not use AT to access the private version field from DataFixer?